testCommon/src/test/java/com/virtuslab/gitmachete/testcommon/TestProcessUtils.java

```diff
@@ -36,8 +36,47 @@ public static String runProcessAndReturnStdout(Path workingDirectory, int timeou
     return stdout;
   }
 
+  @SneakyThrows
+  public static String runGitMacheteCommandAndReturnStdout(Path workingDirectory, int timeoutSeconds, String... arguments) {
+    String[] gitMachete = {"python", "-m", "git_machete.cmd"};
+    int sizeOfGitMachete = gitMachete.length;
+    int sizeOfArguments = arguments.length;
+    int sizeOfNewGitMachete = sizeOfGitMachete + sizeOfArguments;
+    gitMachete = Arrays.copyOf(gitMachete, sizeOfNewGitMachete);
+    for (int i = 0; i < sizeOfArguments; i++) {
+      gitMachete[sizeOfNewGitMachete + (i - sizeOfArguments)] = arguments[i];
+    }
+
+    Process process = new ProcessBuilder()
```

Comment:
Hmm I think this can be simplified away to a call of `runProcessAndReturnStdout`! just with the right params passed as `command`

Reply:
This way you'll avoid massive code repetition here

Reply:
Please note that varargs is just a _syntactic sugar_ (aka _lukier składniowy_) over passing an array: https://stackoverflow.com/a/2925314/2750812